src/ClientData/FunctionInfo.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ClientData/FunctionInfo.h"
+
+#include <absl/strings/match.h>
+#include <absl/strings/str_join.h>
+#include <xxhash.h>
+
+#include "ClientData/ModuleData.h"
+#include "ClientData/ProcessData.h"
+#include "ObjectUtils/Address.h"
+
+namespace orbit_client_data {
+
+namespace {
+uint64_t StringHash(const std::string& string) {
+  return XXH64(string.data(), string.size(), 0xBADDCAFEDEAD10CC);
```

Comment:
What's the purpose of having this in a separate function? As far as I can see it's only called once.